### PR TITLE
CAT-1438 Fix SetLookBrickTest

### PR DESCRIPTION
**Pull request by loibi93**
It was requested that i fix the UserConceptTest too, but unfortunately i don't have a clue why this one is failing. I fixed the SetLookBrickTest, so it would be better to make a new Issue for the other test, maybe someone else has a fix for it.

### CODE REVIEW

**Comment by avihric** [comment]
UserConceptTest has already been fixed in #1303

**Comment by loibi93** [comment]
Sorry, i ment UserBrickScriptActivityTest !
As mentioned here: https://jira.catrob.at/browse/CAT-1438

**Comment by loibi93** [comment]
ok to test

**Comment by loibi93** [comment]
ok to test

**Comment by loibi93** [comment]
Rerun of failed test:
https://jenkins.catrob.at/view/All-Categories/view/Catroid/job/Catroid-CustomEmulator-Single-ClassAndPackage-Test/462/

File: catroidTest/src/org/catrobat/catroid/uitest/content/brick/SetLookBrickTest.java [hunk 1]
```diff
@@ -213,16 +212,14 @@ public void testAddNewLook() {
 
 		LookFragment lookFragment = (LookFragment) currentActivity.getFragment(ScriptActivity.FRAGMENT_LOOKS);
 		lookFragment.startActivityForResult(intent, LookController.REQUEST_SELECT_OR_DRAW_IMAGE);
-
 		solo.waitForActivity(ScriptActivity.class.getSimpleName());
 		solo.goBack();
-		solo.waitForFragmentByTag(LookFragment.TAG);
-
-		solo.sleep(3000);
+		//This is needed, because the spinner is only updated, when you actually click on the dialog
```
Comment (thmq) [inline catroidTest/src/org/catrobat/catroid/uitest/content/brick/SetLookBrickTest.java]: why did you add comments to the code?

**Comment by loibi93** [comment]
Just to make that clear, because normally you shouldn't do it that way !

**Comment by loibi93** [comment]
ok to test

**Comment by loibi93** [comment]
ok to test

**Comment by loibi93** [comment]
I had a look at the two tests that failed on my last run here, and as you can see in Jenkins History, they fail on other Branches two, there seems to be a problem with those two tests !
